### DISCUSSION TITLE
Edge Information `Function`- not showing in UI

### DISCUSSION THREAD

**Discussion by aamir5599** [open-metadata/OpenMetadata#15793] [Q&A]
I am using Openmetadata Version 1.3.0

When a new lineage edge is added through the respective api, its gets added, and also I could fetch the same back and verify if its added properly. 

However, while checking the UI, the lineage is showing up, but the value of `Function` field which is added in the lineagedetails Edge Information is not showing up in the UI.

Previously, I have checked this version 1.2.5, where it is still working fine. 

Can you guide If I have to do it in other way, or should I raise a bug if it is so

TIA

**Comment by harshach** [comment]
@karanh37 can you check above ^

**Comment by karanh37** [comment]
hey @aamir5599 , do you have any screenshot of which `Function` you are referring here.

**Reply by aamir5599** [reply]
Hi, 
First sorry for the delayed reponse. 

I was talking about the function field in the Edge Information which pops up when we click on the lineage edge.
<img width="354" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/151627551/99444d50-5ac5-4fbb-bcd6-43a3ea8ba386">

This value is added via the API
![image](https://github.com/open-metadata/OpenMetadata/assets/151627551/4a683961-2ef3-4865-8fc3-723fae7cc5ae)



The concern is, it is not showing in the UI in  1.3.0, but when I fetch the lineage details via API I could see the added information present there.

**Comment by aamir5599** [comment]
@karanh37 did you get time to check this. TIA

**Reply by karanh37** [reply]
Hey, it's a bug.
I'll create an issue to track and will raise a fix.

**Reply by karanh37** [reply]
Here is the reference ticket to track https://github.com/open-metadata/OpenMetadata/issues/16092
